style/layer/boundary.js

```diff
@@ -0,0 +1,161 @@
+var layerBoundaryCountyBg = {
+  id: "boundary_county-bg",
+  type: "line",
+  paint: {
+    "line-color": "rgba(250, 250, 250, 1)",
```

Comment:
Since you're using the same color in multiple places, consider extracting the colors out to the color constants file:
https://github.com/ZeLonewolf/openstreetmap-americana/blob/049c0d142af77b425d5d1d54da7ecab180f3585a/style/constants/color.js#L1-L5

That way you can just change the colors in one place as you play with the style.

For example, in the road layer:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/049c0d142af77b425d5d1d54da7ecab180f3585a/style/layer/road.js#L5